bigflow/log.py

```diff
@@ -158,15 +111,22 @@ def init_workflow_logging(workflow: 'bigflow.Workflow'):
         print("Log config is not provided for the Workflow")
 
 
+
+def _default_gcp_project():
+    _, project = google.auth.default()
+    return project
+
+
 def init_logging(config: LogConfigDict, workflow_id: str):
+
     global _LOGGING_CONFIGURED
     if _LOGGING_CONFIGURED:
         import warnings
         warnings.warn(UserWarning("bigflow.log is already configured - skip"))
         return
     _LOGGING_CONFIGURED = True
 
-    gcp_project_id = config['gcp_project_id']
+    gcp_project_id = config.get('gcp_project_id') or _default_gcp_project()
```

Comment:
In my opinion, we shouldnt do something like this.

Reply:
Could you explain how this might be possible?

Reply:
@iwko ?

Reply:
User may provide config without `gcp_project_id`. His default config in workflow config is `dev` and he expects that if he did not provide `--config` parameter logs will be sent to the `dev` project. But instead of that if `google.auth.default()` will return prod, project logs will go to prod project instead of dev.

Reply:
You are right, it may happen when users run jobs locally (not via cloud composer).

Reply:
Reverted.